tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -832,6 +832,9 @@ public function test_level3_data_bundle() {
 		];
 
 		$bundle_data = $this->wcpay_gateway->bundle_level3_data_from_items( $items );
+
+		$this->assertEquals( $bundle_data->product_description, '2 more items' );
```

Comment:
We prefer `assertSame` to account for type :)

Reply:
+1 for using `assertSame`

Reply:
Sounds good! Added in https://github.com/Automattic/woocommerce-payments/pull/3789/commits/5e88b591f6e296ea651c495b23fb219fa1e08592